recipe/meta.yaml

```diff
@@ -28,6 +28,8 @@ requirements:
     - {{ compiler('cxx') }}
     - cmake >=3.15
     - make            # [unix]
+    - bison           # [unix]
+    - flex            # [unix]
```

Comment:
Bison and flex are libraries or build tools? This section is reserved for build tools.
If a dependency is only needed at build time (e.g. a single header used privately inside a `.c` file), it should still remain in the `host` section.

Reply:
Looking at a few recipe it seems bison is often used in `build` dependency.

Reply:
bison and flex are build tools.